api/v1alpha1/tracing_types.go

```diff
@@ -114,3 +117,16 @@ type RequestHeaderCustomTag struct {
 	// +optional
 	DefaultValue *string `json:"defaultValue,omitempty"`
 }
+
+// ZipkinConfiguration defines optional configuration for the Zipkin tracing provider.
+type ZipkinConfiguration struct {
+	// TraceId_128Bit determines whether a 128bit trace id will be used
+	// when creating a new trace instance.
+	// +optional
+	TraceId128Bit bool `json:"traceId128Bit,omitempty"`
+	// SharedSpanContext determines whether client and server spans will
+	// share the same span context. Defaults to true.
+	// +kubebuilder:default=true
+	// +optional
+	SharedSpanContext bool `json:"sharedSpanContext,omitempty"`
```

Comment:
should we rename this to DisableSharedSpanContext and make the default value to false?

Reply:
Seeing as SharedSpanContext defaults to true in the backend anyway, I think it makes sense to keep the naming of this field consistent.

Reply:
An API should imporve the UX of most of the end users, envoy may use specific values due to backward compatibility.

Reply:
Updated